docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.                PL[i] = np.linalg.norm(core.z_norm(T[i : i + m]) - core.z_norm(T[nn_i : nn_i + m]))
</pre><p>To get profile value from profile index, we can  probably use the function <code>core._idx_to_mp</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
For now, let's keep it as is since this is a naive function.